js/components/portal-dashboard/class-nav.tsx

```diff
@@ -41,6 +41,7 @@ export class ClassNav extends React.PureComponent<IProps> {
           trackEvent={trackEvent}
           HeaderIcon={ClassIcon}
           dataCy={"choose-class"}
+          isWide={false}
```

Comment:
I would make this an optional param so we don't need to add it unless we need it.